pytorch_kinematics/frame.py

```diff
@@ -99,11 +106,18 @@ def get_transform(self, theta):
         dtype = self.joint.axis.dtype
         d = self.joint.axis.device
         if self.joint.joint_type == 'revolute':
-            t = tf.Transform3d(rot=tf.axis_angle_to_quaternion(theta * self.joint.axis), dtype=dtype, device=d)
+            rot = axis_and_angle_to_matrix(self.joint.axis, theta)
+            matrix = torch.eye(4, dtype=dtype, device=d).unsqueeze(0).repeat(theta.shape[0], 1, 1)
```

Comment:
why explicitly create the 4x4 matrix and pass in via the `matrix=matrix` parameter instead of using the `rot=rot` parameter? This logic is duplicated in the Transform3d constructor